### ISSUE TITLE
One Frame One Tab Bar

### ISSUE THREAD

**Issue by c02y** [ema2159/centaur-tabs#162]
Currently, when I open multiple windows inside a frame, every window contains a tab bar, is it able to only show one tab bar inside one frame? Just like some web browsers, when you tile two windows side by side, you only get one single tab bar at the top for the two windows, you will get another tab bar only when you open a new browser window(new **frame** emacs).

**Comment by thiagolopes** [comment]
That looks a great idea! any maintainer knows if is possible to implement?

**Comment by nebhrajani-a** [comment]
This is already a part of Emacs since 27.1: see `tab-bar-mode` [in the manual](https://www.gnu.org/software/emacs/manual/html_node/emacs/Tab-Bars.html). Centaur, as of the time of writing, does not have support for `tab-bar-mode`. I've thought about adding support, but I'm split on whether this should be a separate package (given that `tab-bar-mode` is quite different conceptually than Centaur) or this should be an optional feature within Centaur. 

In fact, since 27.1, Centaur has its own "built-in" competitor, `tab-line-mode`, which Centaur works independently of. Adding support for this is something I plan to do soon, and has higher priority than `tab-bar-mode` support. However, pull requests or further discussion are always welcome.

**Comment by c02y** [comment]
You can provide it as variable.

**Comment by ema2159** [comment]
Is this solved by #122?

**Comment by c02y** [comment]
No, the same old, two bars in two windows inside one emacs frame.

And it introduced a new bug, in emacsclient, the bar is there, but there are no tabs in it.
